README.rst

```diff
@@ -75,7 +75,7 @@ To use with Django:
         'URL': 'http://localhost:8983/solr/',
         'COLLECTION': 'name',
         # any configSet in SOLR_ROOT/server/solr/configsets
-        'CONFIGSET': 'basic_configs' # optional, basic_configs is default
+        'CONFIGSET': '_default' # optional, _default is default
```

Comment:
Please revise to mention both versions in the comment, e.g. something like:
`'CONFIGSET': '_default'  # optional, _default is default as of Solr x; basic_configs is default for Solr 6.x`

Do we know when this default changed? (8.x or 7.x)?

Reply:
i think it changed with 7, fwiw

Reply:
yep, see https://github.com/Princeton-CDH/parasolr/issues/48